bedrock/settings/base.py

```diff
@@ -1302,5 +1302,6 @@ def lazy_email_backend():
     'activist': '014783244707853607354:bbpl3emdsii',
 }
 
-if config('SWITCH_FLASHTALKING', default=DEV, cast=bool):
-    CSP_IMG_SRC += ('servedby.flashtalking.com',)
+if config('SWITCH_TRACKING_PIXEL', default=DEV, cast=bool):
+    CSP_IMG_SRC += ('ad.doubleclick.net',
```

Comment:
Sorry, I totally didn't see the existing example of using a switch already right above this line of code (wasn't sure if it was OK from `base.py`). Thanks!